include/seastar/core/file.hh

```diff
@@ -61,6 +61,9 @@ struct directory_entry {
     sstring name;
     /// Type of the directory entry, if known.
     std::optional<directory_entry_type> type;
+
+    /// Checks if the file is hidden or not
+    bool is_hidden() const noexcept { return name[0] == '.'; }
```

Comment:
probably could use `name.starts_with(".")`, so it can survive the test like
```cxx
directory_entry e;
e.is_hidden();
```

Reply:
Fair point. Fixed